prio/config.c

```diff
@@ -95,13 +99,14 @@ PrioConfig_numDataFields(const_PrioConfig cfg)
 SECStatus
 Prio_init(void)
 {
-  return rand_init();
+  nss_callbk_init(&nss_pointer);
```

Comment:
Prio_init needs to allow the caller to specify the nss_struct, otherwise this is not configurable

Reply:
the tests are written such that they don't allow the user to configure the input. If i were to update it, then it requires some changes in tests and also the pclient itself. It is a random number generator so i think adding configuration won't change anything in the end?

Reply:
I think you are missing the point here. The purpose of this change is to allow prio to be independent of nss. This way prio can be compiled without nss. A consumer of prio such as firefox would then provide the nss functions via pointers passed in when calling Prio_init. Right now you have hard coded the pointers to be to nss inside the call to `nss_callbk_init` which means prio must once again be compiled with nss. `Prio_init` needs th have the signature `Prio_init(nss_struct *nss_pointer)`

Reply:
ok then I will change it, so now updating the tests and other depending files.